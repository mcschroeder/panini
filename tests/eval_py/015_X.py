def f015(s: str):
  if not len(s) == 1:
    raise Exception
  return s[0]
