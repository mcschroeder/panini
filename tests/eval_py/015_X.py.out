f015 : {s:𝕊 | s ∈ .} → 𝕊